tests/network-setups/fabric/dev/docker/docker-compose-ca.yaml

```diff
@@ -10,6 +10,7 @@ networks:
 
 services:
   ca_org1:
+    profiles: ["1-node", "2-nodes"]
```

Comment:
We are always going to start this service, right? If we forget to supply a profile ID, it won't start. So I think it's safer to remove this line.

Reply:
Based on script default value is `1-node` (i.e. if no profile is specified, this value will be taken), so if we start via script it won't be the case that profile ID is empty.

Reply:
I can remove it, if you feel this is redundant, just thought to it'd be verbose.

Reply:
Yeah, I think it'd be better to remove it, so we don't depend on a well-written script. It's more maintainable that way.